.github/workflows/algolia/src/uploadToAlgolia.ts

```diff
@@ -1,8 +1,7 @@
-import { generateAlgoliaJson, uploadAlgoliaData } from ".";
+import { syncAlgoliaIndex } from ".";
 
 const main = async () => {
-  const records = generateAlgoliaJson();
-  await uploadAlgoliaData(records);
+  await syncAlgoliaIndex();
```

Comment:
@kachkaev reduced it to a single function now.

Reply:
Cool, thanks! We can probably keep `const records = generateAlgoliaJson();` in this file and pass `records` to `syncAlgoliaIndex`. Otherwise, our `syncAlgoliaIndex` and `main` are the same.